pliers/extractors/text.py

```diff
@@ -13,9 +14,16 @@
 import pandas as pd
 import nltk
 from nltk.sentiment.vader import SentimentIntensityAnalyzer
-import sys
+
+import re
+import spacy
```

Comment:
Instead of importing directly, the pattern we use is to assign the import using the `attempt_to_import` helper; see the `sklearn_text` example on [this line](https://github.com/tyarkoni/pliers/pull/319/files#diff-7f7f2c63289715b624c2a87196693f96R29). You would do something similar for spacy.